sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/Spark33XShims.scala

```diff
@@ -144,3 +144,5 @@ trait Spark33XShims extends Spark33XFileOptionsShims {
       })
   ).map(r => (r.getClassFor.asSubclass(classOf[Scan]), r)).toMap
 }
+
+trait GpuDeterministicAggregateShim extends Expression
```

Comment:
let us add a comment about fallback on the default definition of `deterministic`